mne/io/brainvision/tests/data/test_format.ahdr

```diff
@@ -0,0 +1,139 @@
+Brain Vision V-Amp Data Header File Version 1.0
```

Comment:
We keep the old `tests/data` files around because they've been there forever, but try not to add new ones. Can you tell me what is different between this file and the existing ones? If we can copy-edit one or two lines (or just the file extension!) of an existing file using a tmp_path it's better than adding a new file due to repo bloat. If we truly need a new file, nowadays we do it by adding to `mne-testing-data`.

I can explain / give examples of either of these processes once we know which option is better...

Reply:
Indeed the only difference is the header as it is created by the V-Amp. Probably it does not need a separate test.
Should i just remove the file?

Reply:
Let's add it to `mne-testing-data` instead:

https://github.com/mne-tools/mne-testing-data#add-or-change-files-in-the-repo-for-use-with-mne

That way we still cover the new code here to handle the differences